Makefile.defs

```diff
@@ -110,10 +110,16 @@ ifeq ($(GO_MAJOR_VERSION_GE_1),1)
     endif
 endif
 
+GO_BUILD = CGO_ENABLED=0 $(GO) build
+GO_BUILD_WITH_CGO = CGO_ENABLED=1 $(GO) build
+
 ifneq ($(RACE),)
     GO_BUILD_FLAGS += -race
     GO_TEST_FLAGS += -race
     GOTEST_COVER_OPTS += -covermode=atomic
+
+	# GO_BUILD becomes GO_BUILD_WITH_CGO as `-race` requires CGO
```

Comment:
nit indentation seems off here, maybe there is a tab here?

Reply:
Good catch, will fix